### PR TITLE
Generating Guitar Model & Controller With Empty CRUD Actions + Presenting Data from the Actual DB

### PR DESCRIPTION
**Pull request by LiorKGOW**
- Generate Guitars controller with empty CRUD actions using rails generate controller
- Generate Guitar model with the first migration using rails generate model
- Adding resourceful routes to guitars controller
- Presenting data from the DB instead of some mock data & Adding some seed to populate the DB with a few guitars

Screenshot using the Guitars found in the seed.rb file: 
(when you enter the localhost on the browser after running the server the application should look like this)
image

### CODE REVIEW

File: test/models/guitar_test.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class GuitarTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (MariaAga) [inline test/models/guitar_test.rb:8]: Please add a test or delete the file

File: test/fixtures/guitars.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (MariaAga) [inline test/fixtures/guitars.yml:1]: Can be deleted

File: app/views/guitars/delete.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h1>Guitars#delete</h1>
```
Comment (MariaAga) [inline app/views/guitars/delete.html.erb:1]: Since we will be using the CRUD actions as api only, the template files for them can be deleted

**Review by MariaAga — CHANGES_REQUESTED** [review]
Tested and works, some auto generated code/files can be removed as in the comments:

File: app/views/guitars/delete.html.erb (hunk 3 above)
Comment (LiorKGOW) [reply app/views/guitars/delete.html.erb:1]: OK I'll remove them 👍🏼

File: test/fixtures/guitars.yml (hunk 2 above)
Comment (LiorKGOW) [reply test/fixtures/guitars.yml:1]: Will do 👍🏼

File: test/models/guitar_test.rb (hunk 1 above)
Comment (LiorKGOW) [reply test/models/guitar_test.rb]: File removed 👍🏼 
Resolving this conversation

**Comment by LiorKGOW** [comment]
@MariaAga @Ron-Lavi 
I have merged this PR with the content of #33 
In addition, I have changed the commits to be more precise and small
Please when you review have a look at each commit individually 🙏🏼  hopefully they would make better sense separately
You can see the top comment of this PR with the descriptions of the commits
Also, added a screenshot after populating the DB with the seed using `bundle exec rake db:seed`, you should see a new guitar when you enter the localhost after running the server of course :)

**Comment by LiorKGOW** [comment]
> Tested and works, some auto generated code/files can be removed as in the comments:

Thank you Maria for the review
I have included the fixes 👍🏼

File: config/routes.rb [hunk 4]
```diff
@@ -3,5 +3,12 @@
 Rails.application.routes.draw do
   root 'home#index'
   get '/guitars', to: 'guitars#index'
+
+  resources :guitars do
+    member do
+      get :delete
```
Comment (MariaAga) [inline config/routes.rb:9]: Why only delete?

Comment (LiorKGOW) [reply config/routes.rb]: Removed 👍🏼

**Review by MariaAga — APPROVED** [review]
tested and went over the code, looks good!

**Comment by LiorKGOW** [comment]
> tested and went over the code, looks good!

Thank you @MariaAga  :)

**Pull request merged**